From 50b48d076ddceef6bb2c27792e19d6c18e305eae Mon Sep 17 00:00:00 2001
From: Tomas Popela <tpopela@redhat.com>
Date: Fri, 27 Mar 2015 14:00:35 +0100
Subject: [PATCH 0425/1800] EHTMLEditorView - Don't try to preserve the text
 blocks when parsing content

It will be enabled again after the 3.16 is branched so we can improve it.
---
 web-extensions/e-html-editor-view-dom-functions.c | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/web-extensions/e-html-editor-view-dom-functions.c b/web-extensions/e-html-editor-view-dom-functions.c
index 747b6f5f39..66059d0f3c 100644
--- a/web-extensions/e-html-editor-view-dom-functions.c
+++ b/web-extensions/e-html-editor-view-dom-functions.c
@@ -38,6 +38,8 @@
 #define HTML_KEY_CODE_SPACE 32
 #define HTML_KEY_CODE_DELETE 46
 
+#define TRY_TO_PRESERVE_BLOCKS 0
+
 /**
  * e_html_editor_view_dom_exec_command:
  * @document: a #WebKitDOMDocument
@@ -3279,7 +3281,7 @@ parse_html_into_paragraphs (WebKitDOMDocument *document,
 	while (next_br) {
 		gboolean local_ignore_next_br = ignore_next_br;
 		gboolean local_preserve_next_line = preserve_next_line;
-		gboolean preserve_block = TRUE;
+		gboolean preserve_block = TRY_TO_PRESERVE_BLOCKS;
 		const gchar *citation = NULL, *citation_end = NULL;
 		const gchar *rest = NULL, *with_br = NULL;
 		gchar *to_insert = NULL;
-- 
2.15.0

